EIPS/eip-5757.md

```diff
@@ -0,0 +1,94 @@
+---
+eip: 5757
+title: Process for Approving External Resources
+description: Requirements and process for allowing new sources of external resources
+author: Sam Wilson (@SamWilsn)
+discussions-to: https://ethereum-magicians.org/t/eip-5757-process-for-approving-external-resources/11215
+status: Draft
+type: Meta
+created: 2022-09-30
+requires: 1
+---
+
+## Abstract
+
+Ethereum improvement proposals (EIPs) occasionally link to resources external to this repository. This document sets out the requirements for sources that may be linked to, and the process for approving a new source.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+### Definitions
+
+ - **Link**: Any method of referring to a resource, including: markdown links, anchor tags (`<a>`), images, citations of books/journals, and any other method of referencing content not in the current resource.
+ - **Resource**: A web page, document, file, or other media that contains content.
+ - **Source**: A publisher/chronicler of resources, like a standards body (eg. w3c) or a system of referring to documents (eg. Digital Object Identifier System).
+
+### Requirements for Sources
+
+Permissible sources **SHOULD NOT** obviously allow redistributing their resources as part of the EIP repository. If a source does obviously allow redistribution of resources, include those resources in an appropriate directory within this repository and link to those copies instead. The common exception to this rule are git repositories under the Ethereum organization.
+
+Permissible sources **MUST** provide a method of uniquely identifying a particular revision of a resource. Examples of such methods may include git commit hashes, version numbers, or publication dates.
+
+Permissible sources **MUST** have a proven history of availability. A source existing for at least ten years and reliably serving resources would be sufficient—but not necessary—to satisfy this requirement.
+
+Permissible sources **MUST NOT** charge a fee for accessing resources.
```

Comment:
I love the intention but I think this block us from many academic publications. ISO standards are also charging fees. 
```suggestion
WHEN available, author **MUST** choose free permissible sources over non-free permissible sources.

WHEN available, author **MUST** choose least restricting copyright licensed sources over more restricted copyright licensed sources 
```

Reply:
I am fine without allowing links to those sources.

Reply:
I strongly disagree. The accessibility of science suffers from non-free publications. One should not have to dox oneself and have a payment method such as a credit or debit card.

What is written is what I'm happy with. I will not currently accept anything less.

Reply:
But one could not distribute books for free due to copyright reason, meaning we can't cite most of the published books. Is this what we intend for?

Reply:
Yes. This would be an unfortunate side effect.